places.go

```diff
@@ -923,3 +931,83 @@ func (resp *PlacePhotoResponse) Image() (image.Image, error) {
 	img, _, err := image.Decode(resp.Data)
 	return img, err
 }
+
+// FindPlaceFromTextInputType is the different types of inputs.
+type FindPlaceFromTextInputType string
+
+// The types of FindPlaceFromText Input Types.
+const (
+	FindPlaceFromTextInputTypeTextQuery   = FindPlaceFromTextInputType("textquery")
+	FindPlaceFromTextInputTypePhoneNumber = FindPlaceFromTextInputType("phonenumber")
+)
+
+// FindPlaceFromTextRequest is the options struct for Find Place From Text API
+type FindPlaceFromTextRequest struct {
+	// The text input specifying which place to search for (for example, a name,
+	// address, or phone number). Required.
+	Input string
+
+	// The type of input. Required.
+	InputType FindPlaceFromTextInputType
+
+	// Fields allows you to select which parts of the returned details structure
+	// should be filled in.
+	Fields []PlaceDetailsFieldMask
+}
+
+func (r *FindPlaceFromTextRequest) params() url.Values {
+	q := make(url.Values)
+
```

Comment:
super nit: remove the spacing in this method?

Reply:
I'd need to fix it in a bunch of methods...